onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportActivity.java

```diff
@@ -16,13 +16,6 @@
 
 package org.onebusaway.android.report.ui;
 
```

Comment:
Looks like changes to this class are just import formatting - I'd prefer you remove this from the changeset.